### ISSUE TITLE
audiconnect.execute_vehicle_action from readme doesn't work 

### ISSUE THREAD

**Issue by oleg-d** [cyr-ius/hass-audiconnect#34]
I am trying to run the start_climatisation action by using audiconnect.execute_vehicle_action as per the readme but it's not working.

Have the actions been changed to something else? if someone could give me an example of the start_climatisation or is it called now climater.

**Comment by wigster** [comment]
It seems that the service names have been changed -- if you go to the DevTools/Services tool you can type in audiconnect in the service field and now you have "refresh_data" and "turn_(on|off)_action". 

However, supplying the VIN no longer works for the services-- you need to supply the HA device_id -- if you edit the service call in the UI you can pick the car there as a device and if you look back in the YAML you will have the id. If you call the refresh_data service, it does update the data from the car as it should. Turning the lock on/off also works fine.

On the other hand, I can't get turn_on/off to do anything else with the car. For example service call to turn on the climater:
```
service: audiconnect.turn_on_action
data:
  vin: b249b8d4<eaten>c0fca0ef
  action: climater
```
Gives in the logs:
```
2023-12-14 09:42:15.537 ERROR (MainThread) [custom_components.audiconnect.services] (406, {'error': {'errorCode': 'climatisation.bs.notacceptable', 'description': 'Not Acceptable'}})
```

**Comment by oleg-d** [comment]
@wigster yup climater not being able to be turned on has been raised here https://github.com/cyr-ius/hass-audiconnect/issues/22

**Comment by oleg-d** [comment]
> However, supplying the VIN no longer works for the services-- you need to supply the HA device_id -- if you edit the service call in the UI you can pick the car there as a device and if you look back in the YAML you will have the id. If you call the refresh_data service, it does update the data from the car as it should. Turning the lock on/off also works fine.

Is your `device_id` that you use for the vin the same as the `csid` value that can be found in the attributes section of an audi sensor inside the integration?

**Comment by wigster** [comment]
No -- it is not that either. It's really the device_id according to HA. You can get it from putting in the template dev tool e.g.:
```
{{device_id('sensor.smurf_tank_level_in_percentage')}}
```
where I've picked any one of the sensors generated by ha-AudiConnect

**Comment by oleg-d** [comment]
> No -- it is not that either. It's really the device_id according to HA. You can get it from putting in the template dev tool e.g.:
> 
> ```
> {{device_id('sensor.smurf_tank_level_in_percentage')}}
> ```
> 
> where I've picked any one of the sensors generated by ha-AudiConnect

thank you so much! appreciate the help

**Comment by oleg-d** [comment]
Added a PR https://github.com/cyr-ius/hass-audiconnect/pull/44 to update the readme to include audiconnect.turn_on_action
